### PR TITLE
First pass

### PR DESCRIPTION
**Pull request by rcuhljr**
Needs tested. To cut-over have someone run a moonbot with these changes for long enough to update all the timers then you can release this. The failure state will be people not updating to latest will get stale moon data once the last old style moonbot is restarted. You can force people to migrate by deleting the old style moon data in the database if you wish. This should cut our moonwatch bandwidth usage in about half.

### CODE REVIEW

File: dependency.lic [hunk 1]
```diff
@@ -1031,9 +1031,9 @@ class MoonWatch
 
   private
 
-  def update_moon(moon, data)
+  def update_moon(body_name, data)
     echo("updating #{moon} with #{data}") if @debug
```
Comment (rcuhljr) [inline dependency.lic:1035]: change moon here to body_name

**Comment by jandersson** [comment]
Not sure what is causing this error
```
>;moonwatch correct debug                                                        
--- Lich: moonwatch active.                                                      
--- Lich: error: no implicit conversion of Fixnum into String                    
        /usr/share/rvm/rubies/ruby-2.3.7/lib/ruby/2.3.0/time.rb:363:in `_parse'  
        /usr/share/rvm/rubies/ruby-2.3.7/lib/ruby/2.3.0/time.rb:363:in `parse'   
--- Lich: moonwatch has exited.                                                  
```

File: moonwatch.lic [hunk 2]
```diff
@@ -90,10 +93,10 @@ end
...
 def update_sun_info(latest_data)
   return if latest_data.nil? || latest_data.empty?
-  sun_data = latest_data['sun']
-  return if sun_data['set'].nil? || sun_data['rise'].nil?
-  set_time = Time.parse(sun_data['set']).localtime
-  rise_time = Time.parse(sun_data['rise']).localtime
+  sun_data = latest_data['s']
+  return if sun_data['s'].nil? || sun_data['r'].nil?
+  set_time = Time.at(sun_data['s']).localtime
+  rise_time = Time.parse(sun_data['r']).localtime
```
Comment (jandersson) [inline moonwatch.lic:99]: This is raising the exception
```
--- Lich: error: no implicit conversion of Fixnum into String                               
        /usr/share/rvm/rubies/ruby-2.3.7/lib/ruby/2.3.0/time.rb:363:in `_parse'             
        /usr/share/rvm/rubies/ruby-2.3.7/lib/ruby/2.3.0/time.rb:363:in `parse'              
```

Comment (jandersson) [reply moonwatch.lic:99]: I changed `parse` to `at` and it seems to be working

**Comment by jandersson** [comment]
Ran overnight and the timers look good

**Pull request merged**